src/App.vue

```diff
@@ -1106,11 +1104,11 @@ export default class App extends Vue {
         // show survey dialog...
         // - if this is an Incorporation Application filing
         // - if the IA Survey ID is configured
-        // - if the cookie doesn't exist (eg, never set, or expired)
+        // - if the survey id hasn't been saved
         if (this.isIncorporationFiling) {
-          if (+sessionStorage.getItem('IA_SURVEY_ID')) {
-            const cookie = Cookies.get(this.IA_SURVEY_KEY, { domain: this.domain })
-            if (!cookie) {
+          if (this.iaSurveyId > 0) {
+            const savedId = +localStorage.getItem('HIDE_SURVEY')
+            if (savedId !== this.iaSurveyId) {
               this.filingSurveyDialog = true
             }
           }
```

Comment:
This updated mechanism also allows us to set (in 1Password/config map/configuration.json) a NEW survey id and have it display even if a previous survey (id) was hidden.